nixos/tests/invidious.nix

```diff
@@ -5,49 +5,75 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     maintainers = [ sbruder ];
   };
 
-  nodes.machine = { config, lib, pkgs, ... }: {
-    services.invidious = {
-      enable = true;
+  nodes = {
+    postgres-tcp = { config, pkgs, ... }: {
+      services.postgresql = {
+        enable = true;
+        package = pkgs.postgresql_14;
```

Comment:
Since the default in nixos is now 15, I don't feel that great to only downgrade it in the tests. Can we apply one of the workaround in the init script instead?

Reply:
I've changed this now to run with the default PostgreSQL 15 and using an init script.